src/viking.js

```diff
@@ -1,11 +1,68 @@
 // Soldier
-class Soldier {}
-
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(name);
```

Comment:
The super keyword is used to use parameters from the parent class, so in this case it is the parameters health and strength the ones that should be passed from the parent Soldier. Not the parameter name.